pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -547,6 +548,16 @@ type (
 // NodeGroupKubeletConfig contains extra config parameters for the kubelet.yaml
 type NodeGroupKubeletConfig map[string]interface{}
 
+// DeepCopy is needed to generate kubernetes types for NodeGroupKubeletConfig
+func (in *NodeGroupKubeletConfig) DeepCopy() *NodeGroupKubeletConfig {
+	if in == nil {
+		return nil
+	}
+	out := new(NodeGroupKubeletConfig)
+	*out = NodeGroupKubeletConfig(runtime.DeepCopyJSON(*in))
```

Comment:
Ah, yes - looks like it just handled `map[string]interface{}` before, and now it wants to use `runtime.DeepCopyJSON` for it.